nautobot/extras/api/nested_serializers.py

```diff
@@ -134,7 +142,7 @@ class Meta:
         fields = ["id", "url", "name", "image"]
 
 
-class NestedJobSerializer(BaseModelSerializer):
+class NestedJobSerializer(WritableNestedSerializer):
```

Comment:
Do Scheduled Job and Job Result need their nested Job Model writable?

Reply:
`Job` *does* support Notes and ObjectChanges, so we need an  `object_type` on NestedJobSerializer.

JobResultSerializer and ScheduledJobSerializer both specify that their associated NestedJobSerializer is `read_only=True`.